osbrain/agent.py

```diff
@@ -1633,22 +1635,25 @@ def send_recv(self, address, message):
         self.send(address, message)
         return self.recv(address)
 
+    def is_running(self):
```

Comment:
We should add a short docstring (i.e.: "Check whether the agent is running."). You might omit the `Returns` section if you want to, although, for consistency, it should be there too.

Reply:
I thought this was too self-explanatory to need a docstring, but OK 😄